files/zh-cn/web/api/streams_api/concepts/index.md

```diff
@@ -2,103 +2,95 @@
 
-在 JavaScript 中，该过程由调用 {{domxref("ReadableStream.tee()")}} 实现 —— 它返回一个数组，包含对原始可读流的两个相同的副本可读流，可以独立地被不同的 reader 读取。
+在 JavaScript 中，该过程由调用 {{domxref("ReadableStream.tee()")}} 实现——它返回一个数组，包含对原始可读流的两个相同的副本可读流，然后可以独立的使用不同的 reader 读取。
 
-举例而言，您在 [ServiceWorker](/zh-CN/docs/Web/API/Service_Worker_API) 中可能会用到该方法，当您从服务器 fetch 资源，得到一个响应的可读流，您可能会想把这个流拆分成两个，一个流入到浏览器，另一个流入到 ServiceWorker 的缓存。由于 response 的 body 无法被消费两次，以及可读流无法被两个 reader 同时读取，您会需要两个可读流副本来实现需求。
+举例而言，你在 [ServiceWorker](/zh-CN/docs/Web/API/Service_Worker_API) 中可能会用到该方法，当你从服务器 fetch 资源，得到一个响应的可读流，你可能会想把这个流拆分成两个，一个流入到浏览器，另一个流入到 ServiceWorker 的缓存。由于 response 的 body 无法被消费两次，以及可读流无法被两个 reader 同时读取，你会需要两个可读流副本来实现需求。
 
 ![](tee.png)
 
-## Writable streams
+## 可写流
 
-一个 **Writable stream (可写流)** 是一个可以写入数据的数据终点，在 JavaScript 中以一个 {{domxref("WritableStream")}} 对象表示。这是 JavaScript 层面对 **underlying sink (底层汇)** 的抽象 —— 底层汇是某个可以写入原始数据的更低层次的 I/O 数据汇。
+一个 **Writable stream（可写流）** 是一个可以写入数据的数据终点，在 JavaScript 中以一个 {{domxref("WritableStream")}} 对象表示。这是 JavaScript 层面对 **underlying sink（底层接收器）** 的抽象——一个更低层次的 I/O 接收器，将原始数据写入其中。
 
-数据由一个 **writer** 写入流中，每次一个 chunk 。chunk 和可读流的 reader 一样可以有多种类型。您可以用任何方式生成要被写入的块；writer 加上相关的代码称为 **producer**。
+数据由一个 **writer** 写入流中，每次只写入一个分块。分块和可读流的 reader 一样可以有多种类型。你可以用任何方式生成要被写入的块；writer 加上相关的代码称为 **producer**。
 
-When a writer is created and starts writing to a stream (an **active writer**), it is said to be **locked** to it. Only one writer can write to a writable stream at one time. If you want another writer to start writing to your stream, you typically need to abort it before you then attach another writer to it.
+当 writer 被创建并开始写入一个流（**active writer**，译者注：该 writer 已处于活动状态）时，我们说，它被 **locked（锁定）** 在该流上。同一时刻，一个 writer 只能写入一个可写流。如果你想要其他 writer 开始写入你的流，在你将 writer 附着到该流之前，你必须先中止上一个 writer。
 
-An **internal queue** keeps track of the chunks that have been written to the stream but not yet been processed by the underlying sink.
+一个 **internal queue** 跟踪已经被写入流的分块但是仍然没有被底层 sink 处理的分块。
 
-There is also a construct you’ll use called a controller — each writer has an associated controller that allows you to control the stream (for example, to abort it if wished).
+另一个你将用到的对象叫做 controller——每个 writer 都有一个关联的 controller，用来控制流（例如，可以将流关闭）。
 
-![](writable_streams.png)
+![Writable streams data flow](writable_streams.png)
 
-You can make use of writable streams using the {{domxref("WritableStream.WritableStream","WritableStream()")}} constructor. These currently have very limited availability in browsers.
+你可以使用 {{domxref("WritableStream.WritableStream","WritableStream()")}} 构造函数使用可写流。这些目前在浏览器中使用非常有限。
 
-## Pipe chains
+## 链式管道传输（Pipe chain）
 
-The Streams API makes it possible to pipe streams into one another (or at least it will do when browsers implement the relevant functionality) using a structure called a **pipe chain**. There are two methods available in the spec to facilitate this:
+Streams API 使用 **链式管道（pipe chain）**的结构将流传输到另一个流已经成为可能。有两种方法可以作用于它：
```

Comment:
After created the PR, please check the pr preview. It's a easy way to check the render problem:

<https://pr7933.content.dev.mdn.mozit.cloud/zh-CN/docs/Web/API/Streams_API/Concepts>